backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1921,6 +1921,28 @@ databaseChangeLog:
             columns:
               - column:
                   name: test_result_delivery
+  - changeSet:
```

Comment:
Are there any concerns with injecting a changeset further up the history like this? I've only ever seen changesets appended to the end of the changelog.

Reply:
It turns out this doesn't matter at all. If you add a new changeset anywhere, Liquibase will figure it out, run it, and append the `databasechangelog` accordingly.

Reply:
Huh, you learn something new every day!